retention/test/test_cohort_analysis.py

```diff
@@ -0,0 +1,135 @@
+from datetime import date
+import unittest
+import cohort_analysis as ca
```

Comment:
Generally we discourage the use of abbreviated variable names as they aren't as obvious as their expanded counterpart. Doubly so for import statements where the variable definition is so far away from the code itself.